charts/postcoder-proxy/values.yaml

```diff
@@ -13,16 +13,12 @@ externalSecrets: &external_secrets_config
   stores: {}
 
 env:
-  APP_ENV: development
```

Comment:
Not sure why we were doing this?

Reply:
I honestly don't remember, I think I was having issues with new relic running despite being disabled so the automated testing for the chart works properly without having to have a key. If the app has been updated to disable itself properly when a key is not present then this line can be removed.

Reply:
Make sense.  It should be setup correctly now.

Reply:
🤞